datasource/hkobserverfactory.js

```diff
@@ -18,6 +19,26 @@ for (let key in classes)
 	clients[C.getType()] = C;
 }
 
+/**
+ * Instantiate an observer client.
+ * This method will the target HKBase which observer configuration it supports (REST or RabbitMQ)
```

Comment:
The sentence seems to be wrong. Perhaps: "This method will query HKBase which observer configuration it supports (...)"?

Reply:
Yes, I'll correct it.